### DISCUSSION TITLE
[question] next release?

### DISCUSSION THREAD

**Discussion by leonjza** [debauchee/barrier#1271] [Q&A] [answered]
Hello!

First of all, thanks so much for this software! It is incredibly useful!

I wanted to know, when is the next tagged release planned? I noticed it has been over a year since v2.3.3 with 187 commits since.

Thanks!

**Comment by shymega** [comment]
Good point/question. Thank you for the praise. I would agree we need another release. I'm not sure what type of release we should make in terms of versioning. I can collate the changes, and write a proper release notice/update changelog though. Will also need to address any bugs and PRs that can be snuck into the release. What do you think, @p12tic - time for a release?

**Comment by dayne** [comment]
Keep it simple and get a release out just to make sure the system is healthy for pushing a new release before trying to pull in PRs/bug fixes. 

Do that initial changelog update, decide what level of a version tweak to make: major, minor, patch.  I'm guessing minor unless something big is listed. Avoid trying to pick up any major PRs for this because if it goes smoothly then revisit if another effort push is warranted to pull in other PRs. 

Is the release process documented somewhere?  Who does what to trigger that?

**Comment by mavack** [comment]
Does the current master compile?, and do you have details of the build environment for those that are happy to tinker? Or is someone else building snapshots somewhere? 

Answered my own question, yes it does build with some errors, but it works, used the build details from synergy-core bonjour, vs2019, qt 5.12.11.

**Comment by leonjza** [comment, marked as answer]
2.4.0 landed. Thank you! ❤️
